deploy/dev/reload.sh

```diff
@@ -3,32 +3,49 @@ set -xeuo pipefail
 
 . utils.sh
 
-# Script for making it easy to make a change locally and redeploy
-pushd ..
-  ./bin/build
-popd
+main() {
+  # Clean-up previous run
+  helm delete $(helm ls -aq)
```

Comment:
This command may fail or interrupt because of connection ? if so please add retry for this.

Reply:
This is just for local development meaning it run on your local machine so I don't anticipate dramatic connection issues. Also we are just removing helm from the machine which I don't believe requires some sort of connection